src/n_body.rs

```diff
@@ -1,144 +1,224 @@
 // The Computer Language Benchmarks Game
 // http://benchmarksgame.alioth.debian.org/
 //
-// contributed by the Rust Project Developers
-// contributed by Matt Brubeck
+// contributed by the packed_simd developers
```

Comment:
Is their license compatible?

Reply:
It's Apache 2/MIT, like Rust (as it's intended to move into `std` at some point). I believe this is compatible.